Makefile

```
@@ -490,3 +490,23 @@ tutorial_video_publish:
 	  --device $(TUTORIAL_DEVICE)
+
+# Uploads external before/after PNGs to an immutable, commit-addressed R2
+# prefix. Requires the same R2_* .env values and boto3 as tutorial publishing.
+PR_SCREENSHOT_PYTHON ?= python3
+PR_SCREENSHOT_SOURCE ?=
+PR_SCREENSHOT_TOPIC ?=
+PR_SCREENSHOT_COMMIT ?= $(shell git rev-parse HEAD)
+PR_SCREENSHOT_ENV ?= .env
+
+.PHONY: pr_screenshots_publish
+pr_screenshots_publish:
```

checkmake (0.3.2)

[warning] Target body for "pr_screenshots_publish" exceeds allowed length of 5 lines (9).

(maxbodylength)